apps/web/src/components/FilterDialog/FilterDialog.tsx

```diff
@@ -39,12 +45,14 @@ const FilterDialog = <TFieldValues extends FieldValues>({
     defaultValues: options?.defaultValues,
```

Comment:
If `options?.defaultValues` is undefined, I think we should use `initialValues` here.

Reply:
I will try but I think this was intentional because whatever is passed in here is what is used when you call `reset()`. Though, there have been some tweaks since then so maybe that is no longer the case 👍

Reply:
Ugh. I spent way too long on trying to understand why this is. I'm tempted to cast it - I'm _pretty_ sure it should work. Or we leave it as is, and change the docstring describing resetValues and initialValues.

Reply:
Okay, I also don't see a huge issue with a cast here since the full value should cover a partial value 😅 

[update default values to fallback to resetValues](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8784/commits/43d05be5f58a5dedc2b134c5811827f075375814)

Reply:
Even a `Partial<TFieldValues>` doesn't work, it seems to specifically want `DefaultValues<TFieldValues>`. I found this old post which seems related, but unhelpful: https://github.com/orgs/react-hook-form/discussions/3713. Casting it should be fine.